dalle2_pytorch/train_configs.py

```diff
@@ -46,14 +86,51 @@ class DiffusionPriorConfig(BaseModel):
     loss_type: str = 'l2'
     predict_x_start: bool = True
     beta_schedule: str = 'cosine'
+    condition_on_text_encodings: bool = True
+
+    class Config:
+            extra = "allow"
```

Comment:
nit: one indentation too many

Reply:
nice catch, i swore i checked like 5 times for indentations and trailing whitespace 😅 😆